### PR TITLE
add special rate limit for CAS

### PR DESCRIPTION
**Pull request by GeorgeJahad**
Per nathan's request last week

### CODE REVIEW

File: recipes/query.rb [hunk 1]
```diff
@@ -145,10 +145,18 @@
...
         'value' => 4000
       },
+      {
+        'id' => 'cas-special',
+        'uri' => '/version/hybrid:94377/*',
+        'uri_regex' => '/v[0-9.]+/(hybrid:94377)/.+',
+        'http_methods' => 'ALL',
+        'unit' => 'MINUTE',
+        'value' => 4000
+      }
```
Comment (dimtruck) [inline recipes/query.rb]: should be a comma here

Comment (GeorgeJahad) [reply recipes/query.rb]: thanks, i noticecd that but forgot to push it

**Comment by GeorgeJahad** [comment]
@dimtruck any idea why the tests are still failing?

**Comment by GeorgeJahad** [comment]
no longer needed. closing

**Pull request closed without merging**